main.tf

```diff
@@ -108,17 +171,32 @@ resource "aws_vpc_endpoint" "s3_vpc_endpoint" {
   vpc_id          = "${aws_vpc.vpc.id}"
   service_name    = "${data.aws_vpc_endpoint_service.s3.service_name}"
   route_table_ids = ["${concat(aws_route_table.public_routetable.*.id, aws_route_table.private_routetable.*.id)}"]
+
+  tags = "${merge(map("Name", format("%s-s3-endpoint", var.environment)),
```

Comment:
@kdelaat the example fails based the addition of the tags

`Error: module.vpc.aws_vpc_endpoint.s3_vpc_endpoint: : invalid or unknown key: tags`

I guess it requires en update of the aws terraform provider. Please can you verify the example. In case we need to update the provide please add a note about this in the Changelog.